src/plugins/htmlEditor.ts

```diff
@@ -1,10 +1,12 @@
-import Config from "olive/config"
-import Modal from "olive/components/modal"
+import Config from "olive/config";
+import Modal from "olive/components/modal";
 
 export default class HtmlEditor {
 
     input: any;
 
+    public static editorConfigPath : string = "/scripts/ckeditor_config.js";
```

Comment:
This is just a public property that let other microservices override their custom config file path.